sequencer/src/persistence/fs.rs

```diff
@@ -408,6 +412,86 @@ impl SequencerPersistence for Persistence {
             },
         )
     }
+    async fn append_quorum_proposal(
+        &mut self,
+        proposal: &Proposal<SeqTypes, QuorumProposal<SeqTypes>>,
+    ) -> anyhow::Result<()> {
+        let view_number = proposal.data.view_number().u64();
+        let dir_path = self.proposals_dir_path();
+
+        fs::create_dir_all(dir_path.clone()).context("failed to create proposals dir")?;
+
+        let file_path = dir_path.join(view_number.to_string()).with_extension("txt");
+        self.replace(
+            &file_path,
+            |_| {
+                // Don't overwrite the file, we want to append since we can get multiple
+                // saved quorum proposals.
```

Comment:
Not for the same view right? I think not overwriting the file is fine, but it's not going to cause us to append, it's just going to keep what was already there

Reply:
Sorry, this was an old comment. It'll write a new entry per view. I can fix this.